R/ttestonesample.R

```diff
@@ -299,6 +306,12 @@ TTestOneSample <- function(jaspResults, dataset = NULL, options, ...) {
 
       confIntEffSize <- sort(confIntEffSize)
     }
+
+    if (optionsList$wantsEffSizeSE){
+      effectSizeVar <- ((1/n)+(d**2 / (2*n)))
+      #Introduction to Meta-Analysis. Michael Borenstein, L. V. Hedges, J. P. T. Higgins and H. R. Rothstein (2009)
```

Comment:
Could you add 
1. The equation number/page to the comment
2. Add the book into the reference section of the help files